tests/pytest/core/test_session.py

```diff
@@ -200,7 +200,8 @@ def test_rate_limit_time_default(app_request):
     t0 = session.rate_limit_time(app_request)
     t1 = int(time.time())
 
-    assert t1 >= t0
+    # rate limit expiry time should be in the future
+    assert t0 >= t1
```

Comment:
Nice catch with fixing this 💥

Reply:
It was only through adding a more sane default in the `settings.py`, and having the test run (and fail!) that I caught it 🙄